Doc/tutorial/errors.rst

```diff
@@ -147,6 +147,36 @@ For example, the following code will print B, C, D in that order::
 Note that if the *except clauses* were reversed (with ``except B`` first), it
 would have printed B, B, B --- the first matching *except clause* is triggered.
 
+When an exception occurs, it may have associated values, also known as the
```

Comment:
This paragraph needed to move up because the next example uses a named except clause. I also reworded it a bit.